classes/local/execution/engine.php

```diff
@@ -508,6 +508,7 @@ public function finalise() {
     public function abort(?\Throwable $reason = null) {
         if (isset($reason)) {
             $message = $reason->getMessage();
+            $message .= PHP_EOL . $reason->debuginfo;
```

Comment:
Would this be better?
```suggestion
            $message = $reason->getMessage() . PHP_EOL . $reason->debuginfo;
```